FAQs.md

```diff
@@ -1,113 +1,135 @@
-- Click the plus(+) button and click on the folder you wish to include or exclude then click select.
+- Access the settings by clicking the 3 dots on the top-right side of the app (and hit "settings").
+- Scroll down to "library" and select the option to include or exclude.
+- Click the plus (+) button, and click on the folder you wish to include or exclude. Click select.
+
 
 11. **How do I remove an included or excluded folder from the list?**
+
 On the include/exclude list, click the 3 dots beside the folder and select the remove option.
 
 
 12. **Do I have to manually scan new tracks into the music player?**
 
-No, Jockey music player automatically scans library for new tracks.
+No, Jockey automatically scans for new tracks.
 
 
 13. **How do I search for a song or playlist?**
 
-To search for a playlist, the search icon can be found at the top bar of the app. Type the song and search.
-
+The search icon can be found at the top of the app. Type the song name.
 
-14. **What is sleep timer?**
 
-The sleep timer is a feature that shuts down a device or an app after a preset time has elapsed. On Jockey music player, this feature is intended to let users listen to their favorite music while they sleep. The player shuts down after time has elapsed.
+14. **What is the "sleep timer"?**
 
+The sleep timer is a feature that shuts down a device or an app after a preset time has elapsed.
+On Jockey, this feature is intended to let users listen to their favorite music while they sleep.
+The player shuts down after the set time has elapsed.
 
-15. **Where is the sleep timer feature found and How can I use the sleep timer feature?**
 
-The sleep timer feature option is only visible on the now playing page when the 3 dots on the top right side is clicked. To use the sleep timer, 
-click on the 3 dots on the top right side
-select sleep timer and move the pointer on the roll to set time. The maximum time is 120 minutes(2 hours)
+15. **Where is the sleep timer feature found and how can I use it?**
 
+The sleep timer feature option is only visible on the "now playing" page when the 3 dots are clicked.
+To use the sleep timer, click on the 3 dots.
+Select sleep timer and move the pointer on the roll to set the time.
+The maximum time is 120 minutes (2 hours).
 
-16. **Does Jockey music player contain ads?**
 
-No, the app doesn't contain any ads or unnecessary pop-ups
+16. **Does Jockey contain ads?**
 
-17. **Do I have to pay for any feature on Jockey music player ?**
+No, the app doesn't contain any ads or unnecessary pop-ups.
+Check the source code to see for yourself.
 
-No, all the features and tools on Jockey music player are available for free.
 
-18. **What device operating system can use Jockey?**
+17. **Do I have to pay for any features?**
 
-Jockey is an android software meaning Jockey is only available to users using an a device running on Android OS. Android 5.0+
+No, all the features and tools are available for free, like they should be.
```

Comment:
This sounds a bit opinion-ey. Maybe be a bit more neutral here? (I'm fine for charming humor and whatnot, but this feels a bit more passive-aggressive than I'd normally go for)